src/runtime/substitution.cc

```diff
@@ -700,8 +700,11 @@ void Graph::reshape_output_tensor(ParallelTensorShape const &desired_shape) {
       assert(desired_degree % current_degree == 0);
       int partition_factor = desired_degree / current_degree;
 
-      Node partition_node = model->get_or_create_repartition_node(
-          output_tensor, i, partition_factor);
+      RepartitionParams params;
+      params.repartition_legion_dim = i;
+      params.repartition_degree = partition_factor;
+      Node partition_node =
+          model->get_or_create_node<Repartition>(output_tensor, params);
```

Comment:
```suggestion
          model->get_or_create_node<Repartition>(output_tensor, {i/*legion_dim*/, partition_factor});
```
Where not ambiguous this might make the code less verbose

Reply:
I like that. Made the change in other places where there were at most 2 params.